doc/releases.rst

```diff
@@ -338,33 +374,16 @@ For each stable release:
 * The `stable releases and backport team <http://tracker.ceph.com/projects/ceph-releases>`_
   publishes ``point releases`` including fixes that have been backported to the stable release.
 
-In the timeline, the life time of a LTS is calculated to be
-approximately 18 months after the month of the first release. For
-instance, Dumpling is published August 2013 and 18 months starting
-September 2013 is February 2015, therefore by March 2015 Dumpling
-should be retired. The lifetime of a release may vary because it
-depend on how quickly the stable releases are published. For instance
-although Dumpling theoritical retirement was March 2015, it was
-extended to May 2015.
-
-Release numbers conventions
----------------------------
-
-The first Ceph release back in Jan of 2008 was 0.1.  That made sense at
-the time.  The versioning scheme did not change until April 2015,
-when 0.94.1 (the first Hammer point release) was published.  To avoid reaching
-0.99 (and 0.100 or 1.00?) we have a new strategy.
-
-* x.0.z - development releases (for early testers and the brave at heart)
-* x.1.z - release candidates (for test clusters, brave users)
-* x.2.z - stable/bugfix releases (for users)
-
-``x`` will start at 9 for Infernalis (``I`` is the 9th letter), making
-our first development release of the 9th release cycle 9.0.0.
-Subsequent development releases will be 9.0.1, 9.0.2, etc.
+In the timeline, the life time of a stable release series is
+calculated to be approximately 18 months after the month of the first
+release.  The lifetime of a release may vary because it depends on how
```

Comment:
Maybe mention that this is because, we maintain backport releases for 2 release cycles, ie, Luminous will be maintained until the first stable release of Nautilus